openlibrary/templates/type/list/view.html

```diff
@@ -112,7 +112,7 @@ <h1>$list.name</h1>
                 remove_seed(list_key, seed, function() {
                     \$(seed_id).remove();
                     // update seed count
-                    \$("#seed-count").html(render_seed_count(get_seed_count()));
+                    \$("#list-items-count").html(render_seed_count(get_seed_count()));
```

Comment:
**Sigh** it did this :P 
![image](https://user-images.githubusercontent.com/6251786/106794483-e9121980-6626-11eb-8442-25a38e583881.png)

Let's use your initial `load` function; that will work better :+1: